resources/views/custom_fields/fieldsets/edit.blade.php

```diff
@@ -1,49 +1,13 @@
-@extends('layouts.default')
+@extends('layouts/edit-form', [
+    'createText' => trans('admin/custom_fields/general.create_fieldset') ,
+    'updateText' => trans('admin/custom_fields/general.update_fieldset'),
+    'formAction' => (isset($item->id)) ? route('fieldsets.update', ['fieldset' => $item->id]) : route('fieldsets.store'),
+])
 
-{{-- Page title --}}
-@section('title')
-  {{ trans('admin/custom_fields/general.create_fieldset') }}
-@parent
-@stop
-
-@section('header_right')
-<a href="{{ URL::previous() }}" class="btn btn-primary pull-right">
-  {{ trans('general.back') }}</a>
-@stop
-
-
-{{-- Page content --}}
-@section('content')
-<div class="row">
-  <div class="col-md-9">
+@section('inputFields')
 
-  {{ Form::open(['route' => 'fieldsets.store', 'class'=>'form-horizontal']) }}
-    <!-- Horizontal Form -->
-    <div class="box box-default">
-      <div class="box-body">
+  @include ('partials.forms.edit.name', ['translated_name' => trans('general.name')])
 
-          <!-- Name -->
-        <div class="form-group {{ $errors->has('name') ? ' has-error' : '' }}">
-          <label for="name" class="col-md-4 control-label">
-            {{ trans('admin/custom_fields/general.fieldset_name') }}
-          </label>
-          <div class="col-md-6">
-            <input class="form-control" type="text" name="name" id="name" value="{{ old('name') }}" required>
-            {!! $errors->first('name', '<span class="alert-msg" aria-hidden="true"><i class="fas fa-times" aria-hidden="true"></i> :message</span>') !!}
-          </div>
-        </div>
 
-      </div> <!-- /.box-body-->
-      <div class="box-footer text-right">
-        <button type="submit" class="btn btn-success"><i class="fas fa-check icon-white" aria-hidden="true"></i> {{ trans('general.save') }}</button>
-      </div>
 
-    </div> <!-- /.box.box-default-->
-    {{ Form::close() }}
-  </div>
-  <div class="col-md-3">
-    <h2>{{ trans('admin/custom_fields/general.about_fieldsets_title') }}</h4>
```

Comment:
Is the loss of this help-text something we want to avoid?

Reply:
Good catch - the template-ified version (edit.blade.php) isn't really designed for the two-column approach, so I just added the help text in as the rescue icon, same as we have it elsewhere.